### ISSUE TITLE
Tiny 'cracks' in arcs.

### ISSUE THREAD

**Issue by spdkils** [pvcraven/arcade#267]
## Bug Report
When trying to do some simple drawings with primitives, there are tiny stratification in the draw_arc_outline.
### Actual behavior:
![image](https://user-images.githubusercontent.com/11712349/44247108-f9a05200-a196-11e8-80b6-0352061b1824.png)

### Expected behavior:
I would expect them to be smooth, no lines.
### Steps to reproduce/example code:

```python
import arcade
from arcade.color import *

SCREEN_WIDTH = 800
SCREEN_HEIGHT = 600

arcade.open_window(SCREEN_WIDTH, SCREEN_HEIGHT, 'Double Rainbow!', resizable=False)

arcade.start_render()
rainbow = [RED, ORANGE, YELLOW, GREEN, CYAN, BLUE, PURPLE]
x = 400
y = 150
width = 300
height = 300
pen = 10
for idx, color in enumerate(rainbow):
    arcade.draw_arc_outline(x, y, width-(idx*pen), height-(idx*pen), color, 0.0, 180.0, pen)

arcade.finish_render()
arcade.run()
```

### What should be added/changed?
Why for are there cracks in my rainbow!

**Comment by spdkils** [comment]
Well after a bunch of googles, and of course assuming it would be a trivial fix... Yeah, it's not really. I mean it sorta is, you can just double draw every arc.

![image](https://user-images.githubusercontent.com/11712349/44294043-776f6680-a246-11e8-9eff-115e8e4d1eed.png)

**Comment by pvcraven** [comment]
Yep, this is a problem. There seems to be a gap between each segment.

**Comment by pvcraven** [comment]
See: https://gamedev.stackexchange.com/questions/60798/opengl-gl-line-strip-produces-arcs-with-visible-line-edges

Looks like it needs to be a set of triangles or quads.

**Comment by spdkils** [comment]
GL_TRIANGLE_STRIP, and then just draw two arcs side by side, with the distance between then the width?

If I draw 2 points, then just wave back and forth on the arch between the inner diameter, and the outer diameter, wouldn't that make an arc of the width.

Seems like its actually not too bad. I'm working up something and testing.

**Comment by spdkils** [comment]
TOTAL HACK! But looks great! (It can't be that easy can it?)
```
gl.glBegin(gl.GL_TRIANGLE_STRIP)
...
...
    for segment in range(start_segment, end_segment + 1):
        theta = 2.0 * 3.1415926 * segment / num_segments

        x = width * math.cos(theta)
        y = height * math.sin(theta)
        gl.glVertex3f(x, y, 0.5)
        x = (width-border_width) * math.cos(theta)
        y = (height-border_width) * math.sin(theta)
        gl.glVertex3f(x, y, 0.5)

```

**Comment by spdkils** [comment]
Well, I removed all the double ups, I also changed the module to use a triangle_stirp as a test.
I'm not sure if the placement is where I would want, and it may have some corner case issues of drawing the arc. Probably needs more testing, but ... it looks good now.


![image](https://user-images.githubusercontent.com/11712349/44313228-46fc0980-a3b9-11e8-9568-dc3d83bf0bee.png)

**Comment by pvcraven** [comment]
Fixed for next release. Using a triangle strip now.

<img width="601" alt="rainbow" src="https://user-images.githubusercontent.com/1463116/45922380-4a8b2080-be90-11e8-892a-9960b8e0d5c2.png">

**Comment by pvcraven** [comment]
Closed with 2.0 release.
